### ISSUE TITLE
Ability to include defaults in the frontmatter

### ISSUE THREAD

**Issue by apnadkarni** [tclssg/tclssg#64]
Currently it is not possible to include files in the header portion making it necessary to edit multiple files when a header change is required. A facility to include files there similar to how it is done in main content would be useful.

E.g.,
```tcl
{
    title "My web page"
    sidebarNoteDisable 0
    blah blah
    !include common_but_not_global_variable_definitions
} 
## blah blah
Page content
```

**Comment by dbohdan** [comment]
Implemented as "presets" in 3d7789fd28510e6dd044f27015cb63a25e96d7d9 for the future v2.0. (Name and functionality subject to change.)

**Comment by dbohdan** [comment]
Done with [presets](https://github.com/tclssg/tclssg/wiki/Presets).
